silkworm/db/snapshot_merger.cpp

```diff
@@ -84,6 +87,14 @@ struct RawSnapshotWordDeserializer : public SnapshotWordDeserializer {
     }
 };
 
+static std::vector<std::shared_ptr<SnapshotBundle>> bundles_in_range(BlockNumRange range, SnapshotRepository& repository) {
+    std::vector<std::shared_ptr<SnapshotBundle>> bundles;
+    for (auto& bundle : repository.view_bundles())
+        if (range.contains(bundle->block_from()) && range.contains(bundle->block_to() - 1))
+            bundles.push_back(bundle);
+    return bundles;
```

Comment:
```suggestion
    std::vector<std::shared_ptr<SnapshotBundle>> bundles;
    for (auto& bundle : repository.view_bundles()) {
        if (range.contains(bundle->block_from()) && range.contains(bundle->block_to() - 1)) {
            bundles.push_back(bundle);
        }
    }
    return bundles;
```
Even if Google C++ Style Guide allows [not to use curly braces for one statement](https://google.github.io/styleguide/cppguide.html#Formatting_Looping_Branching), here we have a loop with one condition inside, so I believe it's better to add braces, which makes the code also more consistent with the rest (see below in `schedule_bundle_cleanup`).